azext_iot/deviceupdate/_help.py

```diff
@@ -260,3 +260,537 @@ def load_deviceupdate_help():
+          text: >
+            az iot device-update device class update -n {account_name} -i {instance_name} --class-id {device_class_id} --friendly-name "EU-region"
+    """
+
+    helps["iot device-update device class delete"] = """
+        type: command
+        short-summary: Delete a device class or device class subgroup.
+        long-summary: >
+          Device classes are created automatically when Device Update-enabled devices are connected to
+          the hub but are not automatically cleaned up since they are referenced by device class subgroups.
+          If all device class subgroups for a target device class are deleted then the device class itself can also be deleted
+          to remove the records from the system and to stop checking the compatibility of the device class with new
+          updates. If a device is ever reconnected its device class will be re-created if it does not exist.
+
+        examples:
+        - name: Delete a device class.
+          text: >
+            az iot device-update device class delete -n {account_name} -i {instance_name} --class-id {device_class_id}
+
+        - name: Delete a device class and skip the confirmation prompt.
+          text: >
+            az iot device-update device class delete -n {account_name} -i {instance_name} --class-id {device_class_id} -y
+
+        - name: Delete a device class subgroup.
+          text: >
+            az iot device-update device class delete -n {account_name} -i {instance_name} --class-id {device_class_id} --group-id {device_group_id}
+    """
+
+    helps["iot device-update device group"] = """
+        type: group
+        short-summary: Device group management.
+        long-summary: >
+          A device group is a collection of devices. Device groups provide a way to scale deployments to many devices.
+          Each device belongs to exactly one device group at a time. A device group is automatically created when a Device Update-enabled
+          device is connected to the hub and reports its properties.
+    """
+
+    helps["iot device-update device group list"] = """
+        type: command
+        short-summary: List device groups within an instance.
+
+        examples:
+        - name: List device groups.
+          text: >
+            az iot device-update device group list -n {account_name} -i {instance_name}
+
+        - name: List device groups in a desired order, in this case by deviceCount.
+          text: >
+            az iot device-update device group list -n {account_name} -i {instance_name} --order-by deviceCount
+    """
+
+    helps["iot device-update device group show"] = """
+        type: command
+        short-summary: Show details about a device group including the best update and update compliance.
+
+        examples:
+        - name: Show a device group.
+          text: >
+            az iot device-update device group show -n {account_name} -i {instance_name} --group-id {device_class_id}
```

Comment:
`--group-id {device_class_id}` should be replaced by `--group-id {device_group_id}`